config/sync/views.view.user_admin_people.yml

```diff
@@ -203,7 +268,7 @@ display:
           exclude: false
           alter:
             alter_text: false
-            text: ''
+            text: '{{ status__value }} '
```

Comment:
What did the status row show before?

Reply:
This is not something I actively changed, so I dont know :')
I could just see that it works now :D